allennlp/data/vocabulary.py

```diff
@@ -134,12 +144,26 @@ class Vocabulary:
         The default is ``("*tags", "*labels")``, so as long as your namespace ends in "tags" or
         "labels" (which is true by default for all tag and label fields in this code), you don't
         have to specify anything here.
+    pretrained_files : ``Dict[str, str]``, optional
+        If provides, this map specifies the path to optional pretrained embedding files for each
+        namespace. This can be used to either restrict the vocabulary to only words which appear
+        in this file, or to ensure that any words in this file are included in the vocabulary
+        regardless of their count, depending on the value of ``pretrained_exclusive``.
+    pretrained_exclusive : bool, optional (default = False)
```

Comment:
I think `pretrained_exclusive` is a bit ambiguous - are you excluding pretrained words, or exclusively using pretrained words?  E.g., we use "end-exclusive" in span prediction models to say we're _not_ including the end index in the `(start, end)` pair, where here you want "pretrained-exclusive" to mean you are _only_ including the pretrained words.

I think something like `only_include_pretrained` (or even `only_include_pretrained_words`) would be a more obvious name here.